polkadex-primitives/src/lib.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg_attr(all(not(target_env = "sgx"), not(feature = "std")), no_std)]
+#![cfg_attr(target_env = "sgx", feature(rustc_private))]
+
+#[cfg(feature = "sgx")]
+use sgx_tstd as std;
+
+use std::vec::Vec;
+
+use codec::{Encode, Decode};
+use sp_core::crypto::AccountId32;
+
+pub type AccountId = [u8;32];
```

Comment:
I'd recommend to use AccountId32 instead of `[u8;32]` for consistency (see comment above)

Reply:
See my comment above